src/js/components/award/shared/activity/ContractGrantActivityChart.jsx

```diff
@@ -303,10 +303,13 @@ const ContractGrantsActivityChart = ({
     ]);
     // sets the line values - hook - runs on mount and dates change
     useEffect(() => {
-        setStartLineData(Object.assign({}, startLineData, { value: lineHelper(dates._startDate) }));
-        setEndLineData(Object.assign({}, endLineData, { value: lineHelper(dates._endDate) }));
-        setPotentialEndLineData(Object.assign({}, potentialEndLineData, { value: lineHelper(dates._potentialEndDate) }));
-    }, [dates]);
+        if (xDomain?.length) {
+            setStartLineData(Object.assign({}, startLineData, { value: lineHelper(dates._startDate, xDomain) }));
+            setTodayLineData(Object.assign({}, todayLineData, { value: lineHelper(moment(Date.now()), xDomain) }));
+            setEndLineData(Object.assign({}, endLineData, { value: lineHelper(dates._endDate, xDomain) }));
```

Comment:
I wish that `lineHelper` had a more descriptive name. No idea what this function is doing right now.